contracts/error/Errors.sol

```diff
@@ -55,6 +55,7 @@ library Errors {
     error InvalidFeeReceiver(address receiver);
     error InvalidOracleSigner(address signer);
     error InvalidHoldingAddress(address account);
+    error InvalidWithdrawalAmount(uint256 amount);
```

Comment:
could change this to EmptyPositionImpactWithdrawalAmount, i think "amount" param is not needed since it would always be zero